community/samples/serving/helloworld-java-quarkus/README.md

```diff
@@ -0,0 +1,452 @@
+                   <native.image.path>${project.build.directory}/${project.build.finalName}-runner</native.image.path>
+                   <quarkus.http.test-port>8080</quarkus.http.test-port>
+                 </systemProperties>
+               </configuration>
+             </execution>
+           </executions>
+         </plugin>
+         <plugin>
+           <groupId>io.quarkus</groupId>
+           <artifactId>quarkus-maven-plugin</artifactId>
+           <version>${quarkus.version}</version>
+           <executions>
+             <execution>
+               <goals>
+                 <goal>build</goal>
+               </goals>
+             </execution>
+           </executions>
+         </plugin>
+         <plugin>
+           <groupId>org.codehaus.mojo</groupId>
+           <artifactId>build-helper-maven-plugin</artifactId>
+           <version>${build-helper-maven-plugin.version}</version>
+           <executions>
+             <execution>
+               <id>add-resource</id>
+               <phase>generate-resources</phase>
+               <goals>
+                 <goal>add-resource</goal>
+               </goals>
+               <configuration>
+                 <resources>
+                   <resource>
+                     <directory>${project.basedir}/src/main/knative</directory>
+                     <targetPath>${project.build.directory}/knative</targetPath>
+                     <filtering>true</filtering>
+                   </resource>
+                 </resources>
+               </configuration>
+             </execution>
+           </executions>
+         </plugin>
+       </plugins>
+     </build>
+     <dependencies>
+       <dependency>
+         <groupId>io.quarkus</groupId>
+         <artifactId>quarkus-resteasy</artifactId>
+       </dependency>
+       <dependency>
+         <groupId>io.quarkus</groupId>
+         <artifactId>quarkus-junit5</artifactId>
+         <scope>test</scope>
+       </dependency>
+       <dependency>
+         <groupId>io.rest-assured</groupId>
+         <artifactId>rest-assured</artifactId>
+         <scope>test</scope>
+       </dependency>
+     </dependencies>
+     <profiles>
+       <profile>
+         <id>native</id>
+         <build>
+           <plugins>
+             <plugin>
+               <groupId>io.quarkus</groupId>
+               <artifactId>quarkus-maven-plugin</artifactId>
+               <version>${quarkus.version}</version>
+               <executions>
+                 <execution>
+                   <goals>
+                     <goal>native-image</goal>
+                   </goals>
+                   <configuration>
+                     <enableHttpUrlHandler>true</enableHttpUrlHandler>
+                   </configuration>
+                 </execution>
+               </executions>
+             </plugin>
+           </plugins>
+         </build>
+       </profile>
+     </profiles>
+   </project>
+   ```
+
+1. Run the application locally:
```

Comment:
This should be a new section because its no longer about "recreating the code". Maybe "Locally testing your sample"

Reply:
this is also inline w/ other samples w/ precedence. https://github.com/knative/docs/tree/master/docs/serving/samples/hello-world/helloworld-java-spring#recreating-the-sample-code